src/main/java/RedisSessionManagerT8.java

```diff
@@ -0,0 +1,309 @@
+     * @param session
+     * @return
+     */
+    private Hashtable<String, Object> getMetadata(Session session){
+        Hashtable<String, Object> metadata = new Hashtable<>();
+        metadata.put(METADATA_VALID, String.valueOf(session.isValid()));
+        metadata.put(METADATA_CREATION_TIME, DateFormatUtils.ISO_DATE_FORMAT.format(new Date(session.getCreationTime())));
+        metadata.put(METADATA_LAST_ACCESS_TIME, DateFormatUtils.ISO_DATE_FORMAT.format(new Date(session.getLastAccessedTime())));
+        metadata.put(METADATA_MAX_INACTIVE_INTERVAL, String.valueOf(session.getMaxInactiveInterval()));
+        return metadata;
+    }
+
+    /**
+     * Get attributes from a Session
+     * @param session
+     * @return
+     */
+    private Hashtable<String, Object> getAttributes(Session session){
+        RedisSessionT8 redisSession = (RedisSessionT8)session;
+        Hashtable<String, Object> attributes = new Hashtable();
+        for (Enumeration<String> enumerator = redisSession.getAttributeNames(); enumerator.hasMoreElements();) {
+            String key = enumerator.nextElement();
+            attributes.put(key, redisSession.getAttribute(key));
+        }
+        return attributes;
+    }
+
+    //------------------------------------------------------------------------------------------------------------------
+    // END Utils
+    //------------------------------------------------------------------------------------------------------------------
+
+
+    //------------------------------------------------------------------------------------------------------------------
+    // START REDIS
+    //------------------------------------------------------------------------------------------------------------------
+
+    /**
+     * Method to saved session in redis
+     * @param session
+     * @return
+     * @throws IOException
+     */
+    protected boolean saveSession(Session session) throws IOException {
+        boolean errorSaving = true;
+        //Get the session Id
+        String id = session.getId();
+        Hashtable<String, Object> metadata = getMetadata(session);
+        Hashtable<String, Object> attributes = getAttributes(session);
+        try{
+
+            byte[] encodedMetadata = Base64.getEncoder().encode(SerializationUtils.serialize(metadata));
+            byte[] encodedAttributes = Base64.getEncoder().encode(SerializationUtils.serialize(attributes));
+
+            withRedis(
+                    (Jedis jedis)-> {
+                        jedis.setex((id + REDIS_ATTRIBUTES_KEY).getBytes(), session.getMaxInactiveInterval(), encodedAttributes);
+                        jedis.set((id + REDIS_METADATA_KEY).getBytes(), encodedMetadata);
+                        return 0;
+                    }
+            );
+
+            errorSaving = false;
+        }catch (Exception e){
+            log.error("Error - Context: saveToRedis. Description: " + e.getMessage());
+        }
+        return errorSaving;
```

Comment:
You're never checking this return value. This is why return values to report errors is a bad practice. You just wrote this code, and even you aren't using it right.